tools/copyright_checker.js

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env -S deno run --unstable --allow-read --allow-run
+// Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.
+
+import { getSources, ROOT_PATH } from "./util.js";
+
+async function checkCopyright() {
+  const sourceFiles = await getSources(ROOT_PATH, [
+    // js and ts
+    "*.js",
+    "*.ts",
+    ":!:.github/mtime_cache/action.js",
+    ":!:cli/tests/testdata/**",
+    ":!:cli/bench/testdata/**",
+    ":!:cli/tsc/dts/**",
+    ":!:cli/tsc/*typescript.js",
+    ":!:cli/tsc/compiler.d.ts",
+    ":!:test_util/wpt/**",
+    ":!:tools/**", // these files are starts with `#!/usr/bin/env`
+    ":!:cli/tools/init/templates/**",
+
+    // rust
+    "*.rs",
+    ":!:ops/optimizer_tests/**",
+
+    // toml
+    "*Cargo.toml",
+  ]);
+
+  if (!sourceFiles.length) {
+    return;
+  }
+
+  let totalCount = 0;
+  for (const file of sourceFiles) {
```

Comment:
looks like `sourceFiles` includes duplicated items? If I tried to change the copyright headers, the same item appeared twice.

```shellsession
$ ./tools/copyright_checker.js 
Copyright is not sign: /Users/kt3k/denoland/deno/Cargo.toml
Copyright is not sign: /Users/kt3k/denoland/deno/Cargo.toml
Copyright is not sign: /Users/kt3k/denoland/deno/cli/lsp/testing/execution.rs
Copyright is not sign: /Users/kt3k/denoland/deno/cli/lsp/testing/execution.rs
```

Maybe we should do `new Set(sourceFiles)` here?